go.mod

```diff
@@ -1,10 +1,6 @@
 module github.com/grafana/mimir
 
-go 1.23.8
-
-// Please note that this directive is ignored when building with the Mimir build image,
-// that will always use its bundled toolchain.
-toolchain go1.24.2
+go 1.24.2
```

Comment:
`go mod tidy` consistently removes `toolchain` :(

Reply:
I think toolchain is redundant if the version is the same as the Go version.

Reply:
Yes, I'm just unhappy to lose the explanatory comment for when `toolchain` gets reintroduced.